instagram/src/components/CommentSection/AddComment.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+
+
+
+class AddComment extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            text: ''
```

Comment:
I would like to see an array of comments in this component level state that was passed down in props. You can then use this prop comment array to pass in new comment objects into it.